native-image/microservices/micronaut-hello-rest-maven-layered/run.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -ex
+
+./mvnw --no-transfer-progress clean install -Dpackaging=native-image -Pbase-layer
+./mvnw --no-transfer-progress clean package -Dpackaging=native-image -Papp-layer
```

Comment:
the 2nd clean will remove the target directory created as part of the previous install command. Is this intentional?

Reply:
Yes, each build has its own custom target dir.